edgedb/transaction.py

```diff
@@ -39,22 +39,11 @@ class BaseTransaction:
                  '_state', '_nested', '_id', '_managed')
 
     def __init__(self, connection, isolation, readonly, deferrable):
```

Comment:
Do we need to fix nested transactions? Ie see if the isolation level/readonly flag/deferred flag diverge from what is set for the outer block?